cmd/provisioner/createlv.go

```diff
@@ -315,7 +314,7 @@ func createLVS(ctx context.Context, vg string, name string, size uint64, lvmType
 		return "", fmt.Errorf("unsupported lvmtype: %s", lvmType)
 	}
 
-	tags := []string{"lv.metal-stack.io/csi-lvm"}
+	tags := []string{"lv.metal-stack.io/csi-lvm", "isBlock=" + strconv.FormatBool(blockMode)}
```

Comment:
could probably simplified to:
```golang
if blockMode {
 tags = append(tags, "isBlock")
}
```

Reply:
This was meant to be a kind of a safety check for the rare case, that we encounter a volume at reboot, which has been created by a v0.4.x version. If it has no tag at all we can't be sure if it's a normal volume or a block volume that hasn't been examined and tagged before. Currently, these would not get mounted.
The alternative is, to assume it's a normal volume and mount it as a filesystem. Since block volumes currently aren't used anywhere in production, that assumption could be safe too.

Reply:
yes understood.